lib/bcu.rb

```diff
@@ -48,17 +48,34 @@ def self.process(args)
 
     ohai "Found outdated apps"
     Formatter.print_app_table(outdated, state_info, options)
+    printf "\n"
 
-    if options.dry_run
-      printf "\nDo you want to upgrade %d app%s [y/N]? ", outdated.length, (outdated.length > 1) ? "s" : ""
+    unless options.interactive || options.force_yes
+      printf "Do you want to upgrade %d app%s or enter [i]nteractive mode [y/i/N]? ", outdated.length, (outdated.length > 1) ? "s" : ""
```

Comment:
i'am not sure if this looks better:

```suggestion
      printf "Do you want to upgrade %d app%s (or enter [i]nteractive mode) [y/N/i]? ", outdated.length, (outdated.length > 1) ? "s" : ""
```

```suggestion
      printf "Do you want to upgrade %d app%s (or enter [i]nteractive mode) [y/N(i)]? ", outdated.length, (outdated.length > 1) ? "s" : ""
```


```suggestion
      printf "Do you want to upgrade %d app%s (or enter [i]nteractive mode) [y/N i]? ", outdated.length, (outdated.length > 1) ? "s" : ""
```

Reply:
Honestly, I don't think this is going to make any difference and the current wording is good enough, don't you think? 

Not sure if it is worth postponing the merge of this because of that. Thoughts?